api-connection.html

```diff
@@ -100,7 +100,203 @@ <h2 id="function_newConnection">new Connection(config)</h2>
+{% highlight powershell %}
+# The default endpoint
+$endPoint = "http://169.254.169.254/metadata/identity/oauth2/token"
+# The corresponding api-version for the endpoint
+$apiVersion = "2018-02-01"
+# The Azure AD resource URI of the resource for which a token should be obtained.
+# This could be one of the provide URIs within this link
+# [https://docs.microsoft.com/en-us/azure/active-directory/
+# managed-identities-azure-resources/
+# services-support-managed-identities#
+# azure-services-that-support-azure-ad-authentication]
+# or any other resource URI.
+$resourceURI = "https%3A%2F%2Fmanagement.azure.com%2F"
+$tokenAuthURI = $endPoint + "?resource=$resourceURI&api-version=$apiVersion"
+$tokenResponse = Invoke-RestMethod `
+-Method Get `
+-Headers @{Metadata="true"} `
+-Uri $tokenAuthURI                              
+$access_token = $tokenResponse.access_token
+echo "The managed identities for Azure resources access token is $access_token"             
+{% endhighlight %}
+              <h4>A JavaScript for retrieve the token using 'adal-node':</h4>
+              Here is a <a href="https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-powershell" target="_blank"><i><b>link</b></i></a> for microsoft side doc for retrive the token.</h4>
+{% highlight javascript %}
+const { AuthenticationContext } = require('adal-node');
+let generateToken = () => {
+  // The Azure AD resource URI of the resource for which a token should be obtained.
+  // This could be one of the provide URIs within this link
+  // [https://docs.microsoft.com/en-us/azure/active-directory/
+  // managed-identities-azure-resources/
+  // services-support-managed-identities
+  // #azure-services-that-support-azure-ad-authentication]
+  // or any other resource URI.
+  const spn = "https://database.windows.net/";
+  // Azure AD service endpoint
+  const authorityHostUrl ="https://login.microsoftonline.com/";
+  // You Active Directory domain id and name
+  // You find this information in the Azure portal 
+  // under Azure Active Directory > Properties.
+  // Select Directory ID to see your Tenant ID
+  const tenantId ="<Your Tenant ID>";
+  const tenant ="<Your Tenant Name>";
+  //The Azure AD service endpoint is https://login.microsoftonline.com/TENANT/.
+  //Where TENANT is the Azure AD tenant ID or name.
+  const stsurl   = authorityHostUrl+tenant;
+  // If you have not set up you registered application yet, you should probally
+  // consider start setting it. You can still use 7f98cb04-cd1e-40df-9140-
+  // 3bf7e2cea4db which is a hard coded clint Id value within Tedious for
+  // now. However, this value will be deprecated soon.
+  const clientId = "<You Registered Application Tenant ID>"
+  //Your Azure Active Directory user name
+  //The format will be yourusername@tenantname
+  const azureUserName = "<Your Azure User Name>";
+  // Your Azure Active Directory user account password
```

Comment:
```suggestion
  /* Azure AD service endpoint */
  const authorityHostUrl = "https://login.microsoftonline.com/";
  /* Your Active Directory domain id and name.
  You find this information in the Azure portal 
  under Azure Active Directory > Properties.
  Select Directory ID to see your Tenant ID */
  const tenantId = "<Your Tenant ID>";
  const tenant = "<Your Tenant Name>";
  /* The Azure AD service endpoint is https://login.microsoftonline.com/TENANT/
  where TENANT is the Azure AD tenant ID or name. */
  const stsurl = authorityHostUrl+tenant;
  /* If you have not set up your registered application yet, you should consider 
  doing so. You can still use 7f98cb04-cd1e-40df-9140-3bf7e2cea4db
  which is a hard coded clint Id value within Tedious for now. 
  However, this value will be deprecated soon. */
  const clientId = "<Your Registered Application Tenant ID>"
  /* Your Azure Active Directory username.
  The format will be yourusername@tenantname */
  const azureUserName = "<Your Azure User Name>";
  /* Your Azure Active Directory user account password */
```
multiline comments for aesthetics?